firedrake/utility_meshes.py

```diff
@@ -198,6 +198,110 @@ def PeriodicUnitIntervalMesh(ncells):
     return PeriodicIntervalMesh(ncells, length=1.0)
 
 
+def OneElementThickMesh(ncells, Lx, Ly):
+    """
+    Generate a rectangular mesh in the domain with corners [0,0]
+    and [Lx, Ly] with ncells, that is periodic in the x-direction.
+
+    :arg ncells: The number of cells in the mesh.
+    :arg Lx: The width of the domain in the x-direction.
+    :arg Ly: The width of the domain in the y-direction.
+    """
+
+    left = np.arange(ncells, dtype='int32')
+    right = np.roll(left, -1)
+    cells = np.array([left, left, right, right]).T
+    X = np.arange(1.0*ncells)/ncells*Lx
+    Y = 0.*X
+    coords = np.array([X, Y]).T
+
+    # a line of coordinates, with a looped topology
+    plex = PETSc.DMPlex().createFromCellList(2, cells, coords)
+    mesh = Mesh(plex)
+    mesh.topology.init()
+    cell_numbering = mesh._cell_numbering
+    cell_closure = np.zeros((ncells, 9), dtype=int)
+
+    for e in range(ncells):
+        tc = plex.getTransitiveClosure(e)
```

Comment:
You always `tc[0]` later, so you could just declutter the `[0]` by writing
```python
closure, orient = plex.getTransitiveClosure(e)
```
and then just referencing `closure`, not `orient`.